### ISSUE TITLE
Improve community standards

### ISSUE THREAD

**Issue by dpaiton** [element-fi/elf-simulations#332]
The [Community Standards](https://github.com/element-fi/elf-simulations/community) page has suggestions for what we can add to the repo to improve the ability for others to contribute. I'd say the top priority now is filling out `contributing.md`. There are also lots of suggestions in the [GitHub Best Practices page](https://github.com/element-fi/elf-simulations/community). To keep this issue concrete, I'll name these specific tasks:

- [ ] fill in community standards doc
- [ ] add PR template doc
- [ ] add issue template docs
- [ ] Add a commit template doc
- [ ] Add a code of conduct doc
- [ ] Add refs to the aboev docs in the README
- [ ] Make a new issue with additional contributions

So that this issue doesn't live on forever, let's scope it to the above tasks and then when they're done make a new one with additional tasks. Here are some example additional tasks:

- add [semantic-release](https://github.com/semantic-release/semantic-release) and [dependabot](https://github.com/dependabot) integrations
- add governance docs that outline roles & contributions
- add formal roles to the repo, assign roles to contributors
- Set up graphs & dashboards to view [repo health & popularity](https://opensource.guide/metrics/)

**Comment by wakamex** [comment]
always have `"""` alone on a new line
![image](https://github.com/delvtech/elf-simulations/assets/16990562/fc9c2179-06e8-4c9d-8d53-8e9a80816f3f)

**Comment by wakamex** [comment]
> use `r"""` to start docstrings so that we can render backticks and other sphinxdoc stuff

I disagree, we have a whole lot of docstrings right now that don't start with `r"""`, and I prefer the simpler approach. We can use `r"""` only when necessary (render backticks and sphinx docs).

**Comment by wakamex** [comment]
previous discussion around removing parantheses around tuples: https://github.com/delvtech/elf-simulations/pull/136#discussion_r1088280879
